cmake/sanitize.cmake

```diff
@@ -9,6 +9,16 @@ option (SANITIZE "Enable one of the code sanitizers" "")
 set (SAN_FLAGS "${SAN_FLAGS} -g -fno-omit-frame-pointer -DSANITIZER")
 
 if (SANITIZE)
+
+    if (OS_LINUX)
+        execute_process(COMMAND
+            sysctl --binary kernel.randomize_va_space
+            OUTPUT_VARIABLE RANDOMIZE_VA_SPACE)
+        if (NOT ${RANDOMIZE_VA_SPACE} STREQUAL "0")
+            message (FATAL_ERROR "Sanitizer builds require disabled address space layout randomization. Run 'sudo sysctl kernel.randomize_va_space=0' to disable it")
```

Comment:
`sudo sysctl kernel.randomize_va_space=0` disables ASLR only until the next restart.

Permanent setting:
https://askubuntu.com/questions/318315/how-can-i-temporarily-disable-aslr-address-space-layout-randomization